onelogin_aws_cli/configuration.py

```diff
@@ -68,7 +85,10 @@ def section(self, section_name: str):
         :param section_name: Name of the section in the config file
         :return:
         """
-        if not self.has_section(section_name):
+        if not section_name:
```

Comment:
@drewsonne This is the check I was referring to in regards to default section handling